doc/sphinx/changes.rst

```diff
@@ -14,7 +14,85 @@ Version 8.15
 Summary of changes
 ~~~~~~~~~~~~~~~~~~
 
-Not yet written.
+Coq version 8.15 integrates many bug fixes, deprecations and cleanups as well
+as a few new features. We highlight some of the most impactful changes here:
+
+  - The apply with tactic :ref:`no longer renames arguments <815ApplyWith>`
```

Comment:
Should we tag "apply with"?

Reply:
You mean like this right?
```suggestion
  - The :tacn:`apply with <apply>` tactic :ref:`no longer renames arguments <815ApplyWith>`
```

Reply:
I link @Zimmi48 's suggestion, I must admit I was afraid of sphinx bailing on a missing :tacn: and a local link :) (I'm having trouble building the sphinx doc locally)

Reply:
Sure the link is fine too, but at least make it a `tacn` since it currently doesn't look right: https://coq.gitlab.io/-/coq/-/jobs/1956862579/artifacts/_install_ci/share/doc/coq/sphinx/html/changes.html#summary-of-changes